docs/support-guide.md

```diff
@@ -0,0 +1,259 @@
+# OpenShift Provider Certification Tool - Support Guide
+
+> WIP - this document is working in progress
+
+- [Support Case Check List](#check-list)
+    - [New Support Cases](#check-list-new-case)
+    - [New Executions](#check-list-new-executions)
+- [Setting up the Review Environment](#setup)
+    - [Install tools](#setup-install)
+    - [Download dependencies](#setup-download-dependencies)
+    - [Download Partner Results](#setup-download-results)
+- [Review guide: exploring the failed tests](#review-process)
+    - [Exploring the failures](#review-process-exploring)
+    - [Extracting the failures to local directory](#review-process-extracting)
+    - [Explaning the extracted files](#review-process-explain)
+    - [Review Guidelines](#review-process-guidelines)
+
+
+## Support Case Check List <a name="check-list"></a>
+
+### New Support Cases <a name="check-list-new-case"></a>
+
+Check-list to require when **new** support case has been opened:
+
+- Documentation: Installing Steps containing the flavors/size of the Infrastructure and the steps to install OCP
+- Documentation: Diagram of the Architecture including zonal deployment
+- Archive with Certification results
+- Archive with must-gather
+- Installation Check List (file `use-installation-checklist.md`) with the details of the installation
+
+### New Executions <a name="check-list-new-executions"></a>
+
+The following assets, certification assets, should be updated when certain conditions happen:
+
+- Certification Results
+- Must Gather
+- Install Documentation (when any item/flavor/configuration has been modified)
+
+
+The following conditions require new certification assets:
+
+- The version of OpenShift Container Platform has been updated
+- Any Infrastructure component(s) (e.g.: server size, disk category, ELB type/size/config) or cluster dependencies (e.g.: external storage backend for image registry) have been modified
+
+
+## Review Environment <a name="setup"></a>
+
+### Setting up local environment <a name="setup-install"></a>
+
+> TODO: link with each tool URL
```

Comment:
This is safe to be removed unless you had other tools in mind

Reply:
Done